src/Microsoft.ML.TensorFlow/TensorflowUtils.cs

```diff
@@ -32,7 +32,7 @@ internal static class TensorFlowUtils
         /// </summary>
         internal const string TensorflowUpstreamOperatorsKind = "TensorflowUpstreamOperators";
 
-        internal static DataViewSchema GetModelSchema(IExceptionContext ectx, Graph graph, string opType = null)
+        internal static DataViewSchema GetModelSchema(IExceptionContext ectx, Graph graph, string opType = null, bool treatOutputAsBatched = true)
```

Comment:
Does anyone call this with the default parameter? Or can we just make it a normal parameter if all callers pass it in explicitly?

Reply:
I have refactored it so its now a normal parameter.